tools/test-proxy/transition-scripts/assets_gen.ps1

```diff
@@ -0,0 +1,87 @@
+# Criteria for generating the assets.json file
```

Comment:
No parameters here, but I still think it's worth adding a 

```pwsh
<# 
.DESCRIPTION
BLAH
#>
```

But that is very nitpicky for the fact that this is the first draft for it.

Reply:
It'll be added with the next commit.